<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>321</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>301</width>
    <height>321</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>301</width>
    <height>321</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Senhas para Atendimento</string>
  </property>
  <property name="accessibleDescription">
   <string/>
  </property>
  <widget class="QPushButton" name="btn_prioritario">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="accessibleDescription">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius: 10px;
    color: rgb(255, 255, 255);
    background-color: rgb(28, 113, 216);
}

QPushButton:hover {
    background-color: rgb(98, 160, 234);
    color: rgb(26, 95, 180);
}

QPushButton:pressed {
	background-color: rgb(192, 97, 203);
	color: rgb(255, 255, 255);
}

</string>
   </property>
   <property name="text">
    <string>Atendimento Priorit√°rio</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(28, 113, 216);</string>
   </property>
   <property name="text">
    <string>Senhas para Atendimento</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_normal">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="accessibleDescription">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius: 10px;
    color: rgb(255, 255, 255);
    background-color: rgb(28, 113, 216);
}

QPushButton:hover {
    background-color: rgb(98, 160, 234);
    color: rgb(26, 95, 180);
}

QPushButton:pressed {
	background-color: rgb(192, 97, 203);
	color: rgb(255, 255, 255);
}
</string>
   </property>
   <property name="text">
    <string>Atendimento Normal</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_senha">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>176</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>btn_prioritario</tabstop>
  <tabstop>btn_normal</tabstop>
  <tabstop>txt_senha</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
